src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,15 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <>
```

Comment:
There is no need to use fragments here, because you are returning just one element in your jsx